src/data/tie_breaks/tie_breaks.py

```diff
@@ -740,23 +740,23 @@ class SonnebornBergerTieBreak(TieBreak):
 
     @staticmethod
     def static_name() -> str:
-        return _('Sonnenborn-Berger')
+        return _('Sonneborn-Berger')
 
     @staticmethod
     def static_id() -> str:
-        return 'SONNENBORN_BERGER'
+        return 'SONNEBORN_BERGER'
 
     @staticmethod
     def static_papi_id() -> str:
-        return 'Sonnenborn-Berger'
+        return 'Sonneborn-Berger'
```

Comment:
You must keep the bad spelling here, it's what Papi expects to find in the database